flow/error_definitions.h

```diff
@@ -206,6 +203,17 @@ ERROR( key_not_found, 2400, "Expected key is missing")
 ERROR( json_malformed, 2401, "JSON string was malformed")
 ERROR( json_eof_expected, 2402, "JSON string did not terminate where expected")
 
+// 2500 - disk snapshot based backup errors
```

Comment:
Please add new error codes to documentation/sphinx/source/api-error-codes.rst

Reply:
Is this required for all the error codes in error_definitions.h? I do not see many error codes documented here. Is it that the documentation only for transaction related error codes?

Reply:
I don't actually know what the policy is then - I assumed they were all meant to go there

Reply:
API error codes is mainly meant to document error codes that could bubble out to a user. If an error is strictly handled internally, it probably doesn't need to go on the list.